### PR TITLE
Set product name favicon and title

### PR DESCRIPTION
**Pull request by magmat88**
Merged improved-graphic-recognition into this branch. Please first review pull request about improving graphic recognition.

### CODE REVIEW

File: src/app/components/contentItem/contentTile.component.scss [hunk 1]
```diff
@@ -83,20 +88,26 @@
...
-    @extend %rowCentered;
+    @extend %columnLeft;
     @extend %marginZero;
-    @extend %paletteDarkContrast;
+    color: $color_text-dark;
     font-size: 0.9rem;
-    height: 90px;
-    padding: 15px;
+    height: 100%;
+    padding: 0 0 15px 0;
```
Comment (earendillo) [inline src/app/components/contentItem/contentTile.component.scss:96]: wouldn't padding-bottom: 15px be enough?

File: src/app/components/header/header.component.tsx [hunk 2]
```diff
@@ -23,7 +24,11 @@ export function MainHeader({appName}: MainHeaderProps) {
                         className={`hamburger-icon ${iconVisibilityClass}`}
                     ></div>
                 </div>
+
+                {/* source of free rocket svg: https://pixabay.com/pl/vectors/okr%c4%99t-rakietowy-statek-kosmiczny-303591/ */}
```
Comment (earendillo) [inline src/app/components/header/header.component.tsx:28]: xd

File: src/app/components/mainContent/mainContent.component.scss [hunk 3]
```diff
@@ -4,9 +4,12 @@
 
 .main-content-container {
     @extend %columnSpaceBetween;
-    @extend %paletteMedium;
+    @extend %paletteWhite;
     padding-top: 95px;
     position: relative;
+    @include forLargeScreens($breakpoint-medium) {
+        margin: auto 10vw auto 10vw;
```
Comment (earendillo) [inline src/app/components/mainContent/mainContent.component.scss:11]: margin: auto 10vw; is enough I suppose

File: src/app/images/rocket.svg [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (earendillo) [inline src/app/images/rocket.svg:1]: why do we have same code in 2 different files?

**Review by earendillo — CHANGES_REQUESTED** [review]

File: src/app/components/contentItem/contentTile.component.scss (hunk 1 above)
Comment (magmat88) [reply src/app/components/contentItem/contentTile.component.scss:96]: changed to padding-bottom

File: src/app/components/mainContent/mainContent.component.scss (hunk 3 above)
Comment (magmat88) [reply src/app/components/mainContent/mainContent.component.scss:11]: corrected

**Pull request merged**